src/routes/Manual.svelte

```diff
@@ -104,14 +105,43 @@ $: lfDutyCycle = 50;
 $: hfPeriod = 250;
 $: hfDutyCycle = 50;
 
-$: intensity = 100;
+$: intensity = 100; 
+
+$: single_pulse_block = [0,0,0,0];
+$: hfOn = hfPeriod * (hfDutyCycle/100);
+$: hf_block = [hfOn & 0x000000ff, (hfOn & 0x0000ff00) >> 8, hfPeriod & 0x000000ff,(hfPeriod & 0x0000ff00) >> 8];
+$: lfOn = lfPeriod * (lfDutyCycle/100);
+$: lf_block = [lfOn & 0x000000ff, (lfOn & 0x0000ff00) >> 8, lfPeriod & 0x000000ff, (lfPeriod & 0x0000ff00) >> 8];
```

Comment:
While integrating with the feig reader currently, these blocks are set to 0xff. I think this has something to do with a firmware bug. Since everything works for the actuators without the low frequency settings, this hasn't been a priority fix.

It would be good to add a button to show and use low frequency configs, or hide and assume 0xff. Intensity calculated on [0,100] scale would set low frequency to 0xff, high frequency to something small like 10ms and only modulate the high frequency duty cycle.

Reply:
I added the button(Link) for specifying low frequency per your suggestion. Also added a function that is called when the page is loaded after a link is clicked and fixes the appropriate periods/duty cycles.